### PR TITLE
chore: rename function parameters to mixedCase

### PR DESCRIPTION
**Pull request by CJ42**
# What does this PR introduce?

- Rename some function parameters starting with underscore `_` as spotted by Slither.
- rename LSP6Utils library function `generatePermissionsKeysForController` to --> `generatePermissionsKeysForNewController` for better readability + add Natspec comment above function.

### CODE REVIEW

File: contracts/LSP6KeyManager/LSP6Utils.sol [hunk 1]
```diff
@@ -162,26 +162,41 @@ library LSP6Utils {
...
+     *  - keys[0] = `AddressPermissions[]` (array length)
+     *  - keys[1] = `AddressPermissions[index]` (where to store the controller address)
+     *  - keys[2] = `AddressPermissions:Permissions:<controller>`
+     *
+     * @return values : an array of 3 x data values containing:
+     *  - values[0] = the new array length of `AddressPermissions[]`
+     *  - values[1] = the address of the controller
+     *  - values[2] = the `permissions` passed as param
+     */
+    function generatePermissionsKeysForNewController(
```
Comment (JeneaVranceanu) [inline contracts/LSP6KeyManager/LSP6Utils.sol:191]: I'd rename it to `generatePermissionsKeysForAddress` or even shorter `generatePermissionsKeys` as it appears this function is capable of handling any kind of address (not only controllers) and any bit array of permissions.

Comment (CJ42) [reply contracts/LSP6KeyManager/LSP6Utils.sol:191]: The term "controller" here is to be specific and translate _"an address with some permissions"._
I think "controller" gives more context. "Address" is very generic and it is why we changed we also need to make it more clear in the LSP6 Standard page on docs.lukso.tech.

image

In particular what needs to be clear in the name of the function is that it is for **adding a new controller**. See this line below the `+ 1`

image

Comment (CJ42) [reply contracts/LSP6KeyManager/LSP6Utils.sol:191]: could also be `generateNewControllerPermissionsKeys`.

@YamenMerhi wdyt?

Comment (YamenMerhi) [reply contracts/LSP6KeyManager/LSP6Utils.sol:191]: I don't mind both

**Review by YamenMerhi — APPROVED** [review]

**Comment by CJ42** [comment]
@JeneaVranceanu I went with the name `generateNewPermissionsKeys` for the function, so to take into account your review comment (shorter + simple name) and add the notion that it is for a new controller :)

File: contracts/LSP6KeyManager/LSP6Utils.sol (hunk 1 above)
Comment (JeneaVranceanu) [reply contracts/LSP6KeyManager/LSP6Utils.sol]: Oh, right. The `arrayLength + 1` is indeed making a huge difference here.
Looks like I didn't pay enough attention last time.

**Pull request merged**